app.js

```diff
@@ -0,0 +1,206 @@
+    description:
+      'The Ruby on Rails capstone project is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.',
+    skills: ['ruby', 'rails', 'bootstrap'],
+    image: 'images/wishyacht.png',
+    liveLink: 'see live',
+    sourceLink: 'see source',
+  },
+  {
+    title: 'Companies 3D',
+    info: ['frontend', 'Api', '2022'],
+    description:
+      "this is an informative/numeric web application that shows information about the canadian technical companies and their statistcs like the market cap and the trading status it also has search feature and for each company page it's unique root and responsive website",
+    skills: ['react', 'redux', 'bootstrap'],
+    image: 'images/company.png',
+    sourceLink: 'see source',
+  },
+  {
+    title: 'Space hub',
+    info: ['pair', 'APIs', '2022'],
+    description:
+      'A web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions',
+    skills: ['react', 'redux', 'bootstrap'],
+    image: 'images/space.png',
+    liveLink: 'see live',
+    sourceLink: 'see source',
+  },
+];
+
+const btns = document.querySelectorAll('.work button');
+const bgModal = document.querySelector('.bg-modal');
+
+function popUp(index) {
+  const {
+    title, info, description, skills, image,
+  } = popupCards[index];
+
+  const infoHtml = info
+    .map(
+      (i) => `
+      <li class="canopy">${i}</li>
+      ${
+  i === info[info.length - 1]
+    ? ''
+    : '<li><img src="images/Counter.png" alt="counter" /></li>'
+}
+    `,
+    )
+    .join('');
+
+  const skillsHtml = skills
+    .map((s) => `<li class="highlight font-1">${s}</li>`)
+    .join('');
+
+  bgModal.innerHTML = `
+  <div class="modal-content">
+        <div class="modal-header">
+          <h3 class="tonic">${title}</h3>
+          <span class="close-button">+</span>
+        </div>
+        <ul class="ul-box1 flex0">
+          ${infoHtml}
+        </ul>
+        <a href="#"
+          ><img src="${image}" alt="Snapshoot-Portfolio" class="snapshot"
+        /></a>
+        <div class="flex-99">
+          <div class="flex-98">
+            <p class="description-2 flex0">
+            ${description}
+            </p>
+          </div>
+          <div>
+            <ul class="ul-box2 flex0">
+             ${skillsHtml}
+            </ul>
+            <hr class="solid-1">
+            <div class="buttons">
+              <button type="button" class="live-btn">
+                see live
+                <img src="images/Icon.png" alt="live"
+              </button>
+              <button type="button" class="live-btn">
+                see source
+                <img src="images/github.png" alt="live"
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] For some reason as well, in the mobile version I am still not able to scroll up to see the close button of the popup which means I have to refresh the page for it to close.

**This is what I am seeing:**

[Screencast from 21-10-2022 07:36:03 ASUBUHI.webm](https://user-images.githubusercontent.com/86318284/197112915-61888079-2e2b-4cf3-b904-e24c21bec9ea.webm)

- [ ] [OPTIONAL] I realized that when I scroll down on the desktop view, the popup gets overlapped by other contents from other sections of the page. This is not the desired outcome and can be changed by ensuring that the popup has a z-index higher than everything else and it occupies 100% height and width.

**This is what I am seeing:**

[Screencast from 21-10-2022 07:46:55 ASUBUHI.webm](https://user-images.githubusercontent.com/86318284/197114317-b9ff8aee-da1a-467e-823a-67f5d7df7b7f.webm)